roles/ansible-test-splitter/files/list_changed_targets.py

```diff
@@ -10,6 +10,13 @@
 import re
 from collections import defaultdict
 
+try:
```

Comment:
You can do the same with just `requests` or `urllib`. I would prefer we avoid third party dependency in the CI. Also, I don't think we need this try/except block. It's better if the job fails immediately. This will simplify the troubleshooting.